flytekit/clients/auth/authenticator.py

```diff
@@ -219,21 +222,20 @@ def __init__(
                 "Device Authentication is not available on the Flyte backend / authentication server"
             )
         super().__init__(
-            endpoint=endpoint, header_key=header_key or cfg.header_key, credentials=KeyringStore.retrieve(endpoint)
+            endpoint=endpoint, header_key=header_key or cfg.header_key, credentials=KeyringStore.retrieve(endpoint), http_proxy_url=http_proxy_url
         )
 
     def refresh_credentials(self):
-        resp = token_client.get_device_code(self._device_auth_endpoint, self._client_id, self._audience, self._scope)
+        resp = token_client.get_device_code(self._device_auth_endpoint, self._client_id, self._audience, self._scope, self._http_proxy_url)
         print(
             f"""
 To Authenticate navigate in a browser to the following URL: {resp.verification_uri} and enter code: {resp.user_code}
-OR copy paste the following URL: {resp.verification_uri_complete}
```

Comment:
can we check for this field's existence, and show it if it's there?  Even if it's not part of the standard, it's nice to have.

Reply:
i don't think it should be there. resp.verification_uri + resp.user_code is enough for login.

Reply:
yeah but okta supports it.  and lots of people use okta.  it's not in the standard but i feel it's okay to support a nice to have.

Reply:
@kumare3 wdyt ^^

Reply:
can we just add back and merge?